pkg/daemon/criruntime/factory.go

```diff
@@ -197,7 +197,7 @@ func detectRuntime(varRunPath string) []runtimeConfig {
 		}
 	}
 
-	// containerd
+	// other implements of cri, include containerd, crio
```

Comment:
containerd uses `containerd.Client` in `github.com/containerd/containerd` to pull image. It is the specific API of containerd but not common CRI.

Are you sure other implements of CRI, such as CRI-O, can re-use the containerd client?